Lection2/task1.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import "fmt"
+
+func median(i []int) float64 {
+	// Defining one extra variable > calling array length 3 times
+	var arrLength = len(i) / 2
+
+	if len(i)%2 == 1 {
+		return float64(i[arrLength])
+	} else {
```

Comment:
I don't think we need this `else` here: if the condition is not fulfilled, the first `return` will never be executed.